build-scripts/rosa-sync/rosa_sync.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -xeo pipefail
+
+# This script copies the first AMI that is found to exist in the us-east-1 region in the meta.json
+# to the AWS account configured via the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables.
+# The path to the meta.json file must be passed as the first parameter.
+# An optional second parameter configures whether or not to do a dry run (defaults to false).
+ 
+error-exit() {
+    echo "Error: $*" >&2
+    exit 1
+}
+
+copy_ami() {
+    if [ "$#" -lt 1 ]; then
+        error-exit incorrect parameter count for copy_ami $#
+    fi
+
+    if ! [ -x "$(command -v aws)" ]; then
+        error-exit awscli command not found
+    fi
+
+    if [ -z "$AWS_ACCESS_KEY_ID" ]; then
+        error-exit \$AWS_ACCESS_KEY_ID is empty
+    fi
+    
+    if [ -z "$AWS_SECRET_ACCESS_KEY" ]; then
+        error-exit \$AWS_SECRET_ACCESS_KEY is empty
+    fi
+
+    local AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:-us-east-1}
```

Comment:
nit: Commenting as to why `us-east-1` is the default may be nice. Not required.

Reply:
For ROSA we only need to sync each version only to one region, as the replication to all other regions will happen from there within the ROSA account itself. I'll add a comment!

Reply:
Correct, the AWS Marketplace requires we have the AMI in the us-east-1 region to then share it. The sharing process is separate to this process.